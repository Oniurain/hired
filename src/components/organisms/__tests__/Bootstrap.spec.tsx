import * as React from 'react'
import { mount } from 'enzyme'

import { Bootstrap } from '../'

describe('Bootstrap', () => {
  it('should render', () => {

    const subject = mount(
      <Bootstrap />
    )

    expect(subject).toMatchSnapshot()
  })

})
