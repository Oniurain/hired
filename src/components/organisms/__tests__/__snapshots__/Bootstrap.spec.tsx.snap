// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bootstrap should render 1`] = `
<Bootstrap>
  <ReactCSSTransitionGroup
    transitionAppear={true}
    transitionAppearTimeout={500}
    transitionEnter={true}
    transitionEnterTimeout={500}
    transitionLeave={true}
    transitionLeaveTimeout={500}
    transitionName="entry"
  >
    <ReactTransitionGroup
      childFactory={[Function]}
      component="span"
      transitionAppear={true}
      transitionAppearTimeout={500}
      transitionEnter={true}
      transitionEnterTimeout={500}
      transitionLeave={true}
      transitionLeaveTimeout={500}
      transitionName="entry"
    >
      <span>
        <ReactCSSTransitionGroupChild
          appear={true}
          appearTimeout={500}
          enter={true}
          enterTimeout={500}
          leave={true}
          leaveTimeout={500}
          name="entry"
        >
          <styled.div>
            <div
              className="dEKQPY"
            >
              <Component>
                <styled.div>
                  <div
                    className="jTdryN"
                  >
                    <styled.div>
                      <div
                        className="mFYa"
                      />
                    </styled.div>
                     Loadingâ€¦
                  </div>
                </styled.div>
              </Component>
            </div>
          </styled.div>
        </ReactCSSTransitionGroupChild>
      </span>
    </ReactTransitionGroup>
  </ReactCSSTransitionGroup>
</Bootstrap>
`;
